generator/src/test/java/com/scottlogic/deg/generator/cucumber/LessThan.feature

```diff
@@ -2,12 +2,332 @@ Feature: User can specify that a numeric value is lower than, but not equal to,
+       | foo         |
+       | -2147482624 |
+       | -2147482623 |
+       | -2147482622 |
+       | -2147482621 |
+       | -2147482620 |
+
+@ignore #589: produces big-integer numbers rather than integer values, not sure why
+Scenario: lessThan run against a non contradicting not after should be successful (lessThan 1 AND not afterOrAt 2019-01-01T00:00:00.00)
+     Given foo is less than 1
+       And foo is anything but after or at 2019-01-01T00:00:00.000
+       And the generator can generate at most 5 rows
+     Then the following data should be generated:
+       | foo         |
+       | -2147482624 |
+       | -2147482623 |
+       | -2147482622 |
+       | -2147482621 |
+       | -2147482620 |
+
+#before
+@ignore #589: produces big-integer numbers rather than integer values, not sure why
+Scenario: lessThan run against a non contradicting after should be successful (lessThan 1 AND before 2019-01-01T00:00:00.00)
+     Given foo is less than 1
+       And foo is before 2019-01-01T00:00:00.000
+       And the generator can generate at most 5 rows
+     Then the following data should be generated:
+       | foo         |
+       | -2147482624 |
+       | -2147482623 |
+       | -2147482622 |
+       | -2147482621 |
+       | -2147482620 |
+
+@ignore #589: produces big-integer numbers rather than integer values, not sure why
+Scenario: lessThan run against a non contradicting not after should be successful (lessThan 1 AND not before 2019-01-01T00:00:00.00)
+     Given foo is less than 1
+       And foo is anything but before 2019-01-01T00:00:00.000
+       And the generator can generate at most 5 rows
+     Then the following data should be generated:
+       | foo         |
+       | -2147482624 |
+       | -2147482623 |
+       | -2147482622 |
+       | -2147482621 |
+       | -2147482620 |
+
+#beforeOrAt
+@ignore #589: produces big-integer numbers rather than integer values, not sure why
+Scenario: lessThan run against a non contradicting after should be successful (lessThan 1 AND beforeOrAt 2019-01-01T00:00:00.00)
+     Given foo is less than 1
+       And foo is before or at 2019-01-01T00:00:00.000
+       And the generator can generate at most 5 rows
+     Then the following data should be generated:
+       | foo         |
+       | -2147482624 |
+       | -2147482623 |
+       | -2147482622 |
+       | -2147482621 |
+       | -2147482620 |
+
+@ignore #589: produces big-integer numbers rather than integer values, not sure why
+Scenario: lessThan run against a non contradicting not after should be successful (lessThan 1 AND not beforeOrAt 2019-01-01T00:00:00.00)
+     Given foo is less than 1
+       And foo is anything but before or at 2019-01-01T00:00:00.000
+       And the generator can generate at most 5 rows
+     Then the following data should be generated:
+       | foo         |
+       | -2147482624 |
+       | -2147482623 |
+       | -2147482622 |
+       | -2147482621 |
+       | -2147482620 |
```

Comment:
same as above, not sure on these expected results

Reply:
Ordering again, they would be I think but would be last

Reply:
Ref #594